app/models/topic.rb

```diff
@@ -0,0 +1,3 @@
+class Topic < ApplicationRecord
+  validates :name, presence: true, uniqueness: true
+end
```

Comment:
Missing empty line here. You can configure your IDE to append an empty line at the end of each file on save. I don't really know which editor you are using, but in case it's VSCode, here is the [link to the extension](https://marketplace.visualstudio.com/items?itemName=jmlntw.vscode-ensure-single-final-newline).